api/server/router/container/backend.go

```diff
@@ -55,6 +55,7 @@ type monitorBackend interface {
 	ContainerTop(name string, psArgs string) (*container.ContainerTopOKBody, error)
 
 	Containers(config *types.ContainerListOptions) ([]*types.Container, error)
+	ContainersUsage(ctx context.Context) ([]*types.ContainerUsage, error)
```

Comment:
Naming is consistent with other methods of this sort (I would prefer `ContainerUsage`)